packages/core/src/ExtensionManager.ts

```diff
@@ -181,7 +181,7 @@ export class ExtensionManager {
         let defaultBindings: Record<string, () => boolean> = {}
 
         // bind exit handling
-        if (extension.type === 'mark' && extension.config.exitable) {
+        if (extension.type === 'mark' && getExtensionField<AnyConfig['exitable']>(extension, 'exitable')) {
```

Comment:
Per the observation by: https://github.com/ueberdosis/tiptap/pull/4191#issuecomment-1767021524 we should not ever access extension.config to get a field, it needs to just be accessed through this helper that can resolve through inheritance